src/test/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,14 @@
+package calculator;
+
+class StringCalculator {
+
+    private final NumberExtractor numberExtractor;
+
+    public StringCalculator() {
+        this.numberExtractor = new NumberExtractor();
+    }
+
+    public Integer add(String text) {
+        return numberExtractor.extract(text).sum();
```

Comment:
한 줄에 여러 메서드를 연달아 호출하는 것 보다는 적절히 변수로 추출해주면 조금 더 가독성이 좋아질 것 같아요. 🙂

```suggestion
        Numbers numbers = numberExtractor.extract(text);
        return numbers.sum();
```